src/moin/scripts/migration/moin19/import19.py

```diff
@@ -335,8 +337,12 @@ def __init__(self, item, revno, path):
             meta, data = split_body(content)
         meta.update(editlog_data)
         format = meta.pop('format', self.backend.format_default)
+        if format.startswith('csv'):
+            format = 'csv'  # drop trailing sep character as in "format csv ;"
```

Comment:
why is this? why is it only for csv?

Reply:
the next line is:

meta[CONTENTTYPE] = FORMAT_TO_CONTENTTYPE.get(format, CONTENTTYPE_DEFAULT)

csv is recognized, but "csv ,"  defaults to plain text.

I am not aware of other #format commands that pass a parameter similar to "#format csv ,"

Reply:
ok. if this was a general mechanism, we should split at ";" and then take result[0].strip().